### PR TITLE
StDev - Final

### PR DESCRIPTION
**Pull request by ghost**
Basically everything looks good.

Move backtest.py from extensions/strategies/stdev/backtest.py to zenbot/ then sudo python backtest.py

It's going to increment basically the trend 2 and trend 1 until it finds the best trend in the mean2  < mean1 and stdev2 < mean1 calculation at an interval of trend1trades + 1 and trend2 trades y = (x  * 8.6) +- 0.25

Where x in backtest.py is the amount of trendtrades you want to start from. x + 1 at the bottom of backtest.py is the interval of trades to test (defaulted at increment of 1)

### CODE REVIEW

File: extensions/strategies/stddev/backtest.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+while (x < 1000):
+    A = str(x)
+    C = y * 0.1 
+    L = y - C
+    H = y + C
+    while (H > L):
+        J = round(L)
+        K = int(J)
+        I = str(K)
+        command = 'sudo ./zenbot.sh sim --strategy=stddev --trendtrades_1=' + A + ' --trendtrades_2=' + I + ' --min_periods=1250 --period=100ms --days$
```
Comment (DeviaVir) [inline extensions/strategies/stddev/backtest.py]: why sudo?

File: extensions/strategies/stddev/strategy.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      if (s.lookback[s.options.min_periods]) {
+          for (let i = 0; i < s.options.trendtrades_1; i++) { tl0.push(s.lookback[i].close) }
+          for (let i = 0; i < s.options.trendtrades_2; i++) { tl1.push(s.lookback[i].close) }
+          s.std0 = stats.stdev(tl0) / 2
+          s.std1 = stats.stdev(tl1) / 2
+          s.mean0 = math.mean(tl0)
+          s.mean1 = math.mean(tl1)
+          s.sig0 = s.std0 > s.std1 ? 'Up' : 'Down';
+          s.sig1 = s.mean0 > s.mean1 ? 'Up' : 'Down';
+   }
```
Comment (DeviaVir) [inline extensions/strategies/stddev/strategy.js]: weird indentation starts here

File: extensions/strategies/stddev/strategy.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+},
+    onPeriod: function (s, cb) {
+            if (
```
Comment (DeviaVir) [inline extensions/strategies/stddev/strategy.js]: weird indentation

File: package.json [hunk 4]
```diff
@@ -51,10 +52,13 @@
     "semver": "^5.4.1",
     "simple-xmpp": "^1.3.0",
     "sosa_mongo": "^1.0.3",
+    "stats-lite": "2.1.0",
     "talib": "^1.0.3",
     "timebucket": "^0.4.0",
     "ws": "^3.2.0",
     "zero-fill": "^2.2.3"
+
+
```
Comment (DeviaVir) [inline package.json]: ?

**Review by DeviaVir — CHANGES_REQUESTED** [review]
You'll also need to update the `package-lock.json`:
```
rm -rf package-lock.json && npm i && git add package-lock.json && git commit && git push
```

File: extensions/strategies/stddev/backtest.py (hunk 1 above)
Comment (ghost) [reply extensions/strategies/stddev/backtest.py]: Good point, I'll remove that

File: extensions/strategies/stddev/strategy.js (hunk 2 above)
Comment (ghost) [reply extensions/strategies/stddev/strategy.js]: Should be fixed :)

**Comment by JensvdHeydt** [comment]
Could you please git squash all these commits into a single commit? (https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git)

**Review by DeviaVir — CHANGES_REQUESTED** [review]
still missing updated `package-lock.json`

**Pull request closed without merging**